src/styles/index.scss

```diff
@@ -1,3 +1,226 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
```

Comment:
Checklist item: 'Each BEM-block should be in a new file'. The styles for 'body' should be moved to a separate file because 'body' is not a BEM-block and it should not be in the 'index.scss' file.